server/events/project_command_context_builder.go

```diff
@@ -298,14 +322,30 @@ func getTfVersion(ctx *command.Context, absProjDir string) *version.Version {
 	requiredVersionSetting := module.RequiredCore[0]
 	ctx.Log.Debug("found required_version setting of %q", requiredVersionSetting)
 
-	tflist, _ := lib.GetTFList(mirrorURL, true)
-	constrains, _ := semver.NewConstraint(requiredVersionSetting)
-	versions := make([]*semver.Version, len(tflist))
+	tfVersions, err := terraformClient.ListAvailableVersions(ctx.Log)
+	if err != nil {
+		ctx.Log.Err("Unable to list Terraform versions")
```

Comment:
```suggestion
		ctx.Log.Err("Unable to list Terraform versions, may fallback to default terraform version: %s", err)
```

Reply:
Addressed in https://github.com/runatlantis/atlantis/pull/2843/commits/2a3673833e4d4e6c32cc85e163f34b73ab29ca26